ui/site/src/user.ts

```diff
@@ -9,10 +9,15 @@ lichess.load.then(() => {
     if ($zone.hasClass('loaded')) return;
     $zone.addClass('loaded');
     $noteToggle.find('strong').text('' + $zone.find('.note').length);
-    console.log('load', $zone);
-    $zone.find('form').on('submit', function (this: HTMLFormElement) {
+    $zone.children('form').on('submit', function (this: HTMLFormElement, e: SubmitEvent) {
+      const submitter = e.submitter?.classList;
+      const extraData: Record<string, string> = submitter?.contains('btn-mod-note')
+        ? { mod: 'true' }
+        : submitter?.contains('btn-dox-note')
+        ? { mod: 'true', dox: 'true' }
+        : {};
```

Comment:
would it be better to use `<button name="noteMode" value="{dox|mod|normal}">` in the html form, and to remove the JS?

Reply:
That definitely would be nicer but this doesn't work with `formToXhr`/`new FormData()`. As far as I can tell, there doesn't seem to be a proper way to get the exact form data that's about to be submitted in onSubmit.

Reply:
Although I guess we could add it manually from `event.submitter`. Maybe that's still nicer since it means it'll work correctly if the JS fails.

Reply:
Ok, I made some changes so it's using onClick events now but still using name/value on the buttons.